lib/media/drm_engine.js

```diff
@@ -2230,10 +2231,14 @@ shaka.media.DrmEngine = class {
    * @param {!Map.<string, string>} servers
    * @param {!Map.<string, shaka.extern.AdvancedDrmConfiguration>}
    *   advancedConfigs
+   * @param {!Object.<string, string>} keySystemsMapping
```

Comment:
The description above should mention that this also is the step that applies the key systems mapping.

Reply:
Done!